projects/plugins/jetpack/modules/custom-post-types/testimonial.php

```diff
@@ -1,17 +1,28 @@
-<?php
-
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Register a Testimonial post type and handle displaying it anywhere on the site.
+ *
+ * @package automattic/jetpack
+ * @phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
+ * @phpcs:disable MediaWiki.Usage.NestedFunctions.NestedFunction
+ */
+
+/**
+ * Add a Testimonial CPT, and display it with a shortcode
+ */
 class Jetpack_Testimonial {
 	const CUSTOM_POST_TYPE       = 'jetpack-testimonial';
 	const OPTION_NAME            = 'jetpack_testimonial';
 	const OPTION_READING_SETTING = 'jetpack_testimonial_posts_per_page';
 
-	public $version = '0.1';
-
-	static function init() {
+	/**
+	 * Initialize class.
+	 */
```

Comment:
May as well document the return type while we're adding doc comments.
```suggestion
	 *
	 * @return self
	 */
```